modules/tinymce/src/themes/silver/test/ts/browser/editor/header/InlineHeaderTest.ts

```diff
@@ -1,12 +1,13 @@
-import { Waiter } from '@ephox/agar';
+import { UiFinder, Waiter } from '@ephox/agar';
 import { describe, it } from '@ephox/bedrock-client';
 import { Css, Insert, Remove, SelectorFind, SugarBody, SugarElement } from '@ephox/sugar';
 import { TinyHooks } from '@ephox/wrap-mcagar';
+import { assert } from 'chai';
 
 import Editor from 'tinymce/core/api/Editor';
 
 // TODO TINY-10480: Investigate flaky tests
-describe.skip('browser.tinymce.themes.silver.editor.header.InlineHeaderTest', () => {
+describe('browser.tinymce.themes.silver.editor.header.InlineHeaderTest', () => {
```

Comment:
Please delete the comment if it's no longer relevant

but why was skip _added_ below?

- [ ] remove (or explain) the skips

Reply:
in this PR I didn't resolve the flakyness of this tests, I just moved the `skip` from the describe to the tests to don't skip the new test, maybe I could change the comment in:
```
// TODO TINY-10480: Investigate flaky tests (currently skipped)
```
what do you think about it?

Reply:
But is this flake new caused by this change then it needs to be investigated in this ticket. If it's a random flake not related to this change then a comment on each skip inside the describe would be the way to go.

Reply:
Oh I don't know how I missed that it was moved from the `describe` to each `it`.